### PR TITLE
Bootstrap the extension using `spree extension`

### PR DESCRIPTION
**Pull request by sikachu**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+
+gem "spree", github: "spree/spree", branch: "2-4-stable"
```
Comment (elle) [inline Gemfile]: Should this be a dependency in the gemspec file?

Comment (vinniefranco) [inline Gemfile]: Spree is a little weird in this regard. Yes, it should be here :weary:

Comment (sikachu) [inline Gemfile]: We can't point a dependency to GitHub repository so this has to be here.

**Comment by elle** [comment]
:shipit:

**Comment by vinniefranco** [comment]
:+1:

**Pull request merged**